scripts/background.js

```diff
@@ -1,15 +1,37 @@
 chrome.tabs.onUpdated.addListener((tabId, changeInfo, tab) => {
   let tabUrl = tab.url ?? tab.pendingUrl;
-  let fullUrl = tabUrl.split('?')
-  const queryParams = fullUrl[1]
-  let domainOnlyUrl = fullUrl[0];
   if (
     changeInfo.status === 'complete' &&
     tabUrl &&
-    tabUrl.includes('www.reddit.com')
+    tabUrl.includes('medium.com')
   ) {
-    const tedditUrl = domainOnlyUrl.replace('reddit.com', 'teddit.net')
-    const newUrl = queryParams ? `${tedditUrl}?${queryParams}` : tedditUrl;
-    chrome.tabs.update(tabId, { url: newUrl });
+    try {
+      getCookies();
+      chrome.tabs.sendMessage(tabId, { onMedium: true, error: null });
+    } catch (error) {
+      chrome.tabs.sendMessage(tabId, {
+        onMedium: false,
+        error: `The following error has appeared: ${error}`,
+      });
+    }
   }
 });
+
+const getCookies = async () => {
+  const cookies = await chrome.cookies.getAll({});
+  deleteCookies(cookies);
```

Comment:
Once we have our array of cookies, `deleteCookies` is invoked with cookies as an argument.